### DISCUSSION TITLE
سوال

### DISCUSSION THREAD

**Discussion by MPBCoder** [rastikerdar/vazir-font#310] [Q&A]
سلام وقت بخیر امیدوارم حالتون بهتر شده باشه ....
از نظر خودتون اینکه فونتو از طریق CDN که تو آموزش گفتین  اد کنیم بهتره یا از طریق api گوگل فونت؟ کدوم برای سئو بهتره؟

**Comment by zoghal** [comment]
هر کدوم که سرعت لودش برای شما بالاتره بهتره!
درضمن این موضوع ارتباط خاصی با سئو نداره! به استثنای بحث لود صفحات
